src/test/java/org/openrewrite/staticanalysis/LambdaBlockToExpressionTest.java

```diff
@@ -48,4 +48,31 @@ class Test {
           )
         );
     }
+
+    @Test
+    void simplifyLambdaBlockToExpressionWithComments() {
```

Comment:
Can you add a `@Disabled` annotation and an `@Issue` annotation which contains a link to the issue description this test reproduces? Thanks!

Reply:
ok